conans/client/build/msbuild.py

```diff
@@ -87,6 +87,13 @@ def get_command(self, project_file, props_file_path=None, targets=None, upgrade_
                 self._output.warn("***** The configuration %s does not exist in this solution *****" % config)
                 self._output.warn("Use 'platforms' argument to define your architectures")
 
+        if output_binary_log not None:
+            if isinstance(output_binary_log, bool):
+                if output_binary_log:
+                    command.append(' /bl ')
```

Comment:
The join already takes into account whitespace, not necessary that you add extra ones here.